css/styles.css

```diff
@@ -525,18 +528,18 @@ html::-webkit-scrollbar-thumb {
 
   /* ******************** About Hero ***************************** */
   .about-hero {
```

Comment:
Please ensure that all images contain `alt` attributes.
https://github.com/LuisSalas94/capstone-project-1/blob/d9f3c12ad21f9b2d87bc33460cee77aab952a0ce/index.html#L38